README.md

```diff
@@ -609,6 +662,23 @@ knpu_oauth2_client:
             # whether to check OAuth2 "state": defaults to true
             # use_state: true
 
+        # will create service: "knpu.oauth2.client.paypal"
+        # an instance of: KnpU\OAuth2ClientBundle\Client\Provider\PaypalClient
+        # composer require stevenmaguire/oauth2-paypal
+        paypal:
+            # must be "paypal" - it activates that type!
+            type: paypal
+            # add and configure client_id and client_secret in parameters.yml
+            client_id: %paypal_client_id%
+            client_secret: %paypal_client_secret%
+            # a route name you'll create
+            redirect_route: connect_paypal_check
+            redirect_params: {}
+            # When true, client uses Paypal Sandbox urls.
+            # is_sandbox: ''
```

Comment:
This should be `false` as value by default instead of an empty string

Reply:
`bin/update_readme` uses '' instead of boolean value. I will updated it manually but after next usage to update readme it will be again `is_sandbox: ''`

Reply:
Hm, I thought it should be `false` after `scalarNode()` -> `booleanNode()` tweak

Reply:
I have updated update_readme to set boolean values...please check if it is fine